wordpress-mobile/WordPress-iOS
# [Stats Widgets] Update current Today widget to new design

Ref #12849 

This updates the current Today widget to the new design. Specifically, the "compact" view, which is the only one currently displayed. 

Expanded view will be addressed in a future PR. To facilitate that, I've included some things in this PR that may seem unnecessary, but will be used later:
- Likes and Comments are now saved to the plist, but aren't displayed yet.
- The data (Views and Visitors) is inserted with a `TwoColumnRow` view. This will allow easily adding another data "row" when the view is expanded.
- `initRows` only inits one row. But it will be multiple when expanded, hence the plural instead of singular. Stay tuned!

To test:

---
- Start with a fresh install.
- Add the WP widget to the Today view.
- Verify the unconfigured view looks like this:

![unconfigured_light](https://user-images.githubusercontent.com/1816888/68505981-c9beed80-0225-11ea-8160-01f120051946.jpeg)

![unconfigured_dark](https://user-images.githubusercontent.com/1816888/68505988-cf1c3800-0225-11ea-8a3d-ba3f339aeb1d.jpeg)

---
- Log into the app and go to Stats.
- Select 'Today' in the nav bar and select 'Use this site'.
- Return to the widget.
- Verify the view looks like this. Things to note:
  - The widget display name is now "WORDPRESS TODAY".
  - The site url is displayed at the bottom.

![configured_light](https://user-images.githubusercontent.com/1816888/68506014-dfccae00-0225-11ea-8760-4655430464ee.jpeg)

![configured_dark](https://user-images.githubusercontent.com/1816888/68506019-e3603500-0225-11ea-8c5d-0f814cb56e0b.jpeg)

---
Update release notes:
